### ISSUE TITLE
Countdown is not sync'ed on reconnect

### ISSUE THREAD

**Issue by DaQuirm** [DaQuirm/css-quickdraw-redux#89]
Its state updates after up to a second :pouting_cat:

**Comment by R1ZZU** [comment]
@DaQuirm, @kirilknysh isn't it fixed already?

**Comment by kirilknysh** [comment]
I think for `Game` only

**Comment by DaQuirm** [comment]
No, it's not fixed yet, but I guess this one is not really a major issue: the next tick from the server is handled properly, so the user doesn't see the countdown for < 1s.
